app.js

```diff
@@ -38,12 +38,12 @@ app.use(require('./middlewares'));
 app.use(require('./controllers'));
 
 // Basic 404 handler
-app.use(function(req, res) {
+app.use((req, res) => {
```

Comment:
It doesn. once the other one is merged, you will see only the correct ones here.